loadgen/contract/agent-prepare-loadgen.js

```diff
@@ -391,6 +391,7 @@ export default async function startAgent({
       ]);
 
       const rates = {
+        debtLimit: AmountMath.make(centralBrand, 1_000_000_000n),
```

Comment:
Is this backwards compatible? Aka would having this property provided on revisions before https://github.com/Agoric/agoric-sdk/pull/4948 cause the call to fail?

Reply:
It is. The function reading this object isn't concerned if there are other keys it isn't interested in.